diff --git a/node_modules/highlight.js/lib/languages/sqlpp.js b/node_modules/highlight.js/lib/languages/sqlpp.js
new file mode 100644
index 0000000..17752a3
--- /dev/null
+++ b/node_modules/highlight.js/lib/languages/sqlpp.js
@@ -0,0 +1,72 @@
+module.exports = function(hljs) {
+  var COMMENT_MODE = hljs.COMMENT('--');
+  return {
+    case_insensitive: true,
+    illegal: /[<>{}*]/,
+    contains: [
+      {
+        beginKeywords:
+          'begin end commit rollback savepoint alter create drop rename call ' +
+          'delete do insert select truncate update set show grant ' +
+          'merge describe use explain declare prepare execute ' +
+          'analyze flush ' +
+          'revoke values with',
+        end: /;/, endsWithParent: true,
+        lexemes: /[\w\.]+/,
+        keywords: {
+          keyword:
+           'advise all alter analyze and any array as asc at ' +
+              'begin between binary boolean break bucket build by ' +
+              'call case cast cluster collate collection commit committed connect continue correlated cover create current ' +
+              'database dataset datastore declare decrement delete derived desc describe distinct do drop ' +
+              'each element else end every except exclude execute exists explain ' +
+              'false fetch filter first flatten flush following for force from fts function ' +
+              'golang grant group groups gsi ' +
+              'hash having ' +
+              'if isolation ignore ilike in include increment index infer inline inner insert intersect into is ' +
+              'javascript join ' +
+              'key keys keyspace known ' +
+              'language last left let letting level like limit lsm ' +
+              'map mapping matched materialized merge minus missing ' +
+              'namespace nest nl no not nth_value null nulls number' +
+              'object offset on option options or order others outer over ' +
+              'parse partition password path pool preceding prepare primary private privilege probe procedure public ' +
+              'range raw realm rename respect return returning revoke right role rollback row rows ' +
+              'satisfied savepoint schema scope select self semi set show some start statistics string system ' +
+              'then ties to tran transaction trigger true truncate ' +
+              'unbounded under union unique unknown unnest unset update upsert use user using ' +
+              'validate value valued values via view ' +
+              'when where while window with within work' +
+              'xor',
+          literal:
+            'true false null missing',
+          built_in:
+            'number string array object'
+        },
+        contains: [
+          {
+            className: 'string',
+            begin: '\'', end: '\'',
+            contains: [{begin: '\'\''}]
+          },
+          {
+            className: 'string',
+            begin: '"', end: '"',
+            contains: [{begin: '""'}]
+          },
+          {
+            className: 'string',
+            begin: '`', end: '`'
+          },
+          hljs.C_NUMBER_MODE,
+          hljs.C_BLOCK_COMMENT_MODE,
+          COMMENT_MODE,
+          hljs.HASH_COMMENT_MODE
+        ]
+      },
+      hljs.C_BLOCK_COMMENT_MODE,
+      COMMENT_MODE,
+      hljs.HASH_COMMENT_MODE
+    ]
+  };
+};
